### DISCUSSION TITLE
Which arrays on which objects should have uniqueItems or minItems enforced?

### DISCUSSION THREAD

**Discussion by jacobyavis** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#86] [Suggestions] [answered]

**Comment by JSv4** [comment]
Just adding some notes here for additional arrays we'll want to think about (will add to this as I come across more):

- [ ] ContactInfo - emails and phone_numbers. Do we want to let both of these arrays have 0 elements?

**Comment by JSv4** [comment, marked as answer]
This could be a good project for one of the new working groups. A standards / conventions group is being formed to look at these sorts of questions. I'm going to close this out and label it as something for that group to evaluate.
